tests/test_cageKeeper.py

```diff
@@ -326,8 +312,9 @@ def test_get_underwater_urns(self, mcd: DssDeployment, keeper: CageKeeper, guy_a
         previous_eth_price = open_underwater_urn(mcd, mcd.collaterals['ETH-A'], guy_address)
         open_cdp(mcd, mcd.collaterals['ETH-C'], our_address)
 
+        ilks = keeper.check_ilks()
 
-        urns = keeper.get_underwater_urns()
+        urns = keeper.get_underwater_urns(ilks)
```

Comment:
This is probably fine assuming `UrnHistory` is tested well enough to test the vdb endpoint.

Reply:
I believe `UrnHistory` is used in production within Auction Keepers. 
It does fall back on our former [logFrob](https://github.com/makerdao/pymaker/blob/master/pymaker/dss.py#L353) approach if a vdb endpoint is not provided